addon/helpers/toggle.js

```diff
@@ -1,10 +1,32 @@
 import { helper } from 'ember-helper';
 import get from 'ember-metal/get';
 import set from 'ember-metal/set';
+import { isPresent } from 'ember-utils';
 
-export function toggle([prop, obj]) {
+function nextIndex(length, currentIdx) {
+  let nextIdx;
+
+  if (currentIdx + 1 === length) {
+    nextIdx = 0;
+  } else if (currentIdx !== -1) {
```

Comment:
Can you add a test that does not match either conditional above? (i.e. currentValue is not one of the toggle given states)